repo/db/inventory.go

```diff
@@ -42,7 +42,7 @@ func (i *InventoryDB) Get(slug string) (map[string]int, error) {
 	i.lock.RLock()
 	defer i.lock.RUnlock()
 	ret := make(map[string]int)
-	stm := `select * from inventory where slug like "` + slug + `%";`
+	stm := `select * from inventory where slug="` + slug + `";`
```

Comment:
As long as this function is called without user input it should be safe but I think we should always use parameterization for stuff like this so somebody down the line doesn't pass in user input without realizing how dangerous it is.